atomicapp/providers/kubernetes.py

```diff
@@ -204,3 +205,52 @@ def undeploy(self):
             if self.config_file:
                 cmd.append("--kubeconfig=%s" % self.config_file)
             self._call(cmd)
+
+    def persistent_storage(self, graph, action):
+        """
+        All extra methods that belong to 'requirements'
+        Each method will be passed the respective graph from Nulecule
+        and any other extra information that is required.
```

Comment:
I feel like this description doesn't apply to this function. Are we trying to generically define what requirements functions are? If so I think we need to be more specific or something.